requirements.txt

```diff
@@ -12,11 +12,11 @@ Werkzeug==0.11.8
 factory-boy==2.5.2
 fake-factory==0.5.7
 itsdangerous==0.24
-psycopg2==2.6.1
+psycopg2==2.8.2
```

Comment:
Maybe the binary?

Reply:
Good idea, but when checking the documentation I think it shouldn't be used in prod:

"
The binary package is a practical choice for development and testing but in production it is advised to use the package built from sources.
"
--https://pypi.org/project/psycopg2/

Reply:
Okay, sorry, I thought the binary was the preferred one.